operations/compute_rolling_upgrades/Upgrade_Compute_Nodes_with_CRUS.md

```diff
@@ -121,28 +123,30 @@ Upgrade a set of compute nodes with the Compute Rolling Upgrade Service \(CRUS\)
+    A CRUS session goes through a number of steps \(approximately the number of nodes to be upgraded divided by the requested step size\) to complete an upgrade. Each step
+    moves through the following stages, unless the boot session is interrupted by being deleted.
 
-    1.  Starting - Preparation for the step and CRUS initiates WLM quiescing of nodes.
-    2.  Quiescing - Waits for all WLM nodes in the step to reach a quiesced \(not busy\) state.
-    3.  Quiesced - The nodes in the step are all quiesced and CRUS initiates booting the nodes into the upgraded environment.
-    4.  Booting - Waits for the boot session to complete.
-    5.  Booted - The boot session has completed. Check the success or failure of the boot session. Mark all nodes in the step as failed if the boot session failed.
-    6.  WLM Waiting - The boot session succeeded. Wait for nodes to reach a ready state in the WLM. All nodes in the step that fail to reach a ready state within 10 minutes of entering this stage are marked as failed.
-    7.  Cleanup - The upgrade step has finished. Clean up resources to prepare for the next step.
-    When a step moves from one stage to the next, CRUS adds a message to the messages field of the upgrade session to mark the progress.
+    1. `Starting` - Preparation for the step. CRUS initiates WLM quiescing of nodes.
+    1. `Quiescing` - Waits for all WLM nodes in the step to reach a quiesced \(not busy\) state.
+    1. `Quiesced` - The nodes in the step are all quiesced and CRUS initiates booting the nodes into the upgraded environment.
+    1. `Booting` - Waits for the boot session to complete.
+    1. `Booted` - The boot session has completed. Check the success or failure of the boot session. Mark all nodes in the step as failed if the boot session failed.
+    1. `WLM Waiting` - The boot session succeeded. Wait for nodes to reach a ready state in the WLM. All nodes in the step that fail to reach a ready state within 10 minutes of entering this stage are marked as failed.
+    1. `Cleanup` - The upgrade step has finished. Clean up resources to prepare for the next step.
 
-6.  Optional: Delete the CRUS upgrade session.
+    When a step moves from one stage to the next, CRUS adds a message to the `messages` field of the upgrade session to mark the progress.
 
-    Once a CRUS upgrade session it is complete, it can no longer be used. It can be kept for historical purposes if desired, or it can be deleted.
+1. Delete the CRUS upgrade session. (Optional)
+
+    Once a CRUS upgrade session is completed, it can no longer be used. It can be kept for historical purposes if desired, or it can be deleted.
 
     ```bash
-    ncn-w001# cray crus session delete $UPGRADE_ID
+    ncn# cray crus session delete $UPGRADE_ID
```

Comment:
ncn# doesn't seem right. I don't think this will be something you run from a storage node

Reply:
Addressing this with https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4815